### DISCUSSION TITLE
Searchable select inside Repeater causing error

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#9961] [Help]
Searchable select inside `Repeater` causing error.
**array_key_exists(): Argument #1 ($key) must be a valid array offset type**

**Comment by leandrocfe** [comment]
Could you share the code please?

**Reply by ArtMin96** [reply]
Sorry, I didn't emphasize that the problem is only with the simple repeater.

```php
Repeater::make('name')
    ->simple(
        Select::make('field_name')
            ->searchable(),
    )
```

**Reply by leandrocfe** [reply]
I didn't understand the issue. There isn't problem when using searchable inside repeater.
